google/api/experimental/experimental.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
+
+import "google/api/annotations.proto";
+import "google/api/experimental/authorization_config.proto";
+
+option go_package = "google.golang.org/genproto/googleapis/api;api";
+option java_multiple_files = true;
+option java_outer_classname = "ExperimentalProto";
+option java_package = "com.google.api";
+option objc_class_prefix = "GAPI";
+
+
+// Experimental service configuration. These configuration options can
+// only be used by whitelisted users.
+message Experimental {
```

Comment:
I am not sure if we should put experimental files here. We traditionally do not publish them in common protos packages, due to the semver implications.

Reply:
I did this because service.proto is referring to it and I was, honestly, doing this so hastily I didn't want to risk further brekaage. Should I edit that reference to experimental.proto out? I can do so later.